app/views/editor/cinematic/CinematicEditorComponent.vue

```diff
@@ -26,14 +26,18 @@
         <div class="col-md-4">
           <span>There is no autosave. Please click this button often.</span>
           <button v-on:click="saveCinematic" :disabled="state.saving || !cinematic">save</button>
+          <button v-on:click="runCinematic">Test Cinematic</button>
           <button><a href="/cinematic">Play View (please save first)</a></button>
         </div>
       </div>
 
       <div class="row">
-        <div class="col-md-12">
+        <div class="col-md-6">
           <div id="treema-editor" ref="treemaEditor" v-once></div>
         </div>
+        <div class="col-md-6" v-if="rawData">
+          <cinematic-canvas :cinematicData="rawData" :key="rerenderKey" />
```

Comment:
Here we're attaching the cinematic canvas vue component to the editor to make editing easier.